specifications/models/caffe2/mobilenet_v2/mobilenet_v2_quant.json

```diff
@@ -0,0 +1,30 @@
+{
+  "model": {
+    "category": "CNN",
+    "description": "Trained MobileNet v2 on Caffe2",
+    "files": {
+      "init": {
+        "filename": "model_init.pb",
+        "location": "/Users/pritam/caffe2_mobilenetv2_quant/model_init.pb",
+        "md5": "c503fcfbac53d8a3b4a5b716e898b59e"
+      },
+      "predict": {
+        "filename": "model.pb",
+        "location": "/Users/pritam/caffe2_mobilenetv2_quant/model.pb",
+        "md5": "dc2dba5ef9ed216678558301f4690063"
+      }
+    },
+    "format": "caffe2",
+    "kind": "deployment",
+    "name": "mobilenet_v2_quant"
+  },
+  "tests": [
+    {
+      "arguments": "--net {files.predict} --init_net {files.init} --warmup {warmup} --iter {iter} --input \"data\" --input_dims \"1,3,224,224\" --input_type float --run_individual true",
```

Comment:
Looks good, suggest to change this to "command" : "{program} --net ...."

This is the recommended syntax moving forward